src/pages/Dogs/Dogs.jsx

```diff
@@ -52,10 +53,23 @@
           </div>
         </BreadcrumbBar>
       </div>
-      <AdoptionLogNavbar />
-      {data.map(facility => (
-        <AdoptionLog key={facility.name} tableName={facility.name} tableId={facility.id} />
-      ))}
+      <AdoptionLogNavbar
+        setFacilityFilter={setFacilityFilter}
+        facilityFilter={facilityFilter}
+        setSearchDog={setSearchDog}
+        searchDog={searchDog}
+      />
+      {facilityFilter ? (
+        <div>
+          <AdoptionLog tableName={data[facilityFilter].name} searchDog={searchDog} tableId={facility.id} />
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'facility' is not defined.